kpatch-build/kpatch-build

```diff
@@ -233,6 +233,12 @@ support_klp_replace()
 	fi
 }
 
+kconfig_set()
+{
+	local config_setting="$1"
+	[[ "$config_setting" == "y" || "$config_setting" == "Y" ]]
+}
+
```

Comment:
I think helper function isn't really needed, and IMO hurts readability, when instead we can just do `[[ -n "$CONFIG_PRINTK_INDEX" ]]` or `[[ -z "$CONFIG_DEBUG_INFO" ]]`.

Reply:
@jpoimboe : the last push dropped the helper, look good?